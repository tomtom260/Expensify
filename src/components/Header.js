import React, { Fragment } from 'react';
import { NavLink } from 'react-router-dom';

const Header =()=>(
    <Fragment>
        <h1>Expensify</h1>
        <NavLink exact activeClassName="activeLink" to="/">Home</NavLink>
        <NavLink activeClassName="activeLink" to="/create">Create Expense</NavLink>
    </Fragment>
)

export default Header;