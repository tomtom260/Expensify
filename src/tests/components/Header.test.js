import React from 'react';
import {shallow} from 'enzyme';
// import ReactShallowRenderer from 'react-test-renderer/shallow';
import Header from '../../components/Header';

test('should render Header correctly',()=>{
    // const renderer = new ReactShallowRenderer();
    // renderer.render(<Header/>)
    // expect(renderer.getRenderOutput()).toMatchSnapshot()
    const wrapper = shallow(<Header/>)
    expect(wrapper).toMatchSnapshot()
})