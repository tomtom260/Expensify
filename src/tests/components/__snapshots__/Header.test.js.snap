// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<Fragment>
  <h1>
    Expensify
  </h1>
  <NavLink
    activeClassName="activeLink"
    exact={true}
    to="/"
  >
    Home
  </NavLink>
  <NavLink
    activeClassName="activeLink"
    to="/create"
  >
    Create Expense
  </NavLink>
</Fragment>
`;
